packages/@aws-cdk/aws-cloudfront/lib/origin.ts

```diff
@@ -172,6 +184,20 @@ export abstract class OriginBase implements IOrigin {
     if (path.endsWith('/')) { path = path.substr(0, path.length - 1); }
     return path;
   }
+
+  /**
+   * Takes origin shield region and converts to CfnDistribution.OriginShieldProperty
+  */
+  private getOriginShield(originShieldRegion:string = ''): CfnDistribution.OriginShieldProperty | undefined {
```

Comment:
We often use `render` as the method verb when converting to a Cfn* interface.

```suggestion
  private renderOriginShield(originShieldRegion?: string): CfnDistribution.OriginShieldProperty | undefined {
```